docs/changehistory/NextVersion.md

```diff
@@ -39,10 +41,17 @@ For more details, see the new descriptions of the `texelStepSize` and `maxDistan
 
 ## Transformer API
 
-The synchronous `void`-returning overload of [IModelTransformer.initFromExternalSourceAspects]($transformer) has been deprecated.
-It will still perform the old behavior synchronously until it is removed. It will now however return a `Promise` (which should be
-awaited) if invoked with the an [InitFromExternalSourceAspectsArgs]($transformer) argument, which is necessary when processing
-changes instead of the full source contents.
+The function [IModelTransformer.initFromExternalSourceAspects]($transformer) has been deprecated, in most cases you no longer need to use it.
```

Comment:
Whoever makes the next addon is going to have a mess of conflicts in NextVersion.md. I think all your revisions can go in the "Deprecations" section on master.